<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventAppletFormatConfig</class>
 <widget class="QWidget" name="EventAppletFormatConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>380</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="5">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>The format for the displayed text for events.
Substitutions (mostly self-explanatory) are:
%{startDate}, %{endDate} - start and end date, %{startTime}, %{endTime} - start and end time,
%{summary} - summary, %{description} - description, %{location} - location, %{duration} - duration,
%{yearsSince} - years since the first time (useful for birthdays),
%{resourceName} - the i18n name of the Akonadi resource,
%{mainCategory}, %{categories} - the first or all categories,
%{tab} - a tabulator</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Format for normal events:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="KLineEdit" name="normalEventEdit">
        <property name="toolTip">
         <string>The text displayed for common events.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QTreeWidget" name="categoryFormatWidget">
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Category</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Format</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>281</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="addCategoryButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="copyCategoryButton">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="removeCategoryButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>121</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
